CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ BUG FIXES:
 * Transitive dependencies were lost during apply when the referenced resource expanded into zero instances ([#33403](https://github.com/hashicorp/terraform/issues/33403))
 * OpenTF will no longer override SSH settings in local git configuration when installing modules. ([#33592](https://github.com/hashicorp/terraform/issues/33592))
 * Handle file-operation errors in `internal/states/statemgr`. ([#278](https://github.com/opentffoundation/opentf/issues/278))
+* `opentf init`: OpenTF will no longer allow downloading remote modules to invalid paths. ([#278](https://github.com/opentffoundation/opentf/issues/356))
```

Comment:
Shouldn't we add a link to the original issue in terraform?
The linked issue only references the needed backport, and not the issue that is solved

Moreover, if we do need to keep the current in place - mind that you used a wrong issue number (`278` -> `356`) in the link

Reply:
Why 268? i've used 356

Reply:
I'm fine with referencing our issue, as it have link to the other one

Reply:
```suggestion
* `opentf init`: OpenTF will no longer allow downloading remote modules to invalid paths. ([#356](https://github.com/opentffoundation/opentf/issues/356))